modules/chapter2/pages/section1.adoc

```diff
@@ -1,3 +1,76 @@
-= Section 1
+= Data Science Projects
 
-This is _Section 1_ of _Chapter 2_ in the *hello* quick course....
\ No newline at end of file
+In RHODS, a _data science project_ is the preferred way to organize resources when working on an AI/ML application.
+Internally, a data science project is an OpenShift project that contains specific labels to identify the project as a RHODS-specific project,
+for example:
+
+[source,yaml,subs="+quotes"]
+----
+kind: Project
+apiVersion: project.openshift.io/v1
+metadata:
+  name: my-ai-project
+  labels:
+    kubernetes.io/metadata.name: my-ai-project
+    *modelmesh-enabled: 'true'*
+    *opendatahub.io/dashboard: 'true'*
+    ...
+----
+
+== The Need for Data Science projects
+
```

Comment:
I wonder if this is the right place to introduce definitions from the product docs - https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science_self-managed/1.33/html-single/working_on_data_science_projects/index#working-on-data-science-projects_nb-server

That important note in the product docs is also relevant to the code block you mentioned earlier. please see how to incorporate these into the lecture

Reply:
All these definitions have been already introduced in [ch01s02](file:///home/jairamir/courses/rhods-intro/build/site/rhods-intro/1.33/chapter1/section2.html). 

I will incorporate the note, though